### PR TITLE
Documentation Update

### PR DESCRIPTION
**Pull request by teddyrendahl**
Much needed! 

@hhslepicka can you take special care with the Qt Installation part? I wonder if we should just point this to a common explanation...

### CODE REVIEW

File: docs/source/basic_usage.rst [hunk 1]
```diff
@@ -33,87 +33,153 @@ Interpreting a Device
...
+    import happi
+    from typhon.plugins import register_client
+
+    # Initialize a new JSON based client
+    client = happi.Client(path='db.json', initialize=True)
+    # Register this with typhon
+    register_client(client)
+    # Add a device to our new database
+    device = happi.Device(device_class='ophyd.sim.SynAxis',
+                          prefix='Tst:Mtr', args=[], kwargs={{name}},
```
Comment (klauer) [inline docs/source/basic_usage.rst]: `kwargs={{name}}` ?

File: docs/source/basic_usage.rst [hunk 2]
```diff
@@ -33,87 +33,153 @@ Interpreting a Device
...
+It is crucial that we understand the importance of these signals to the
+operator. In order to glean this information from the object the ``kind``
+attributes are inspected. For more information see the `ophyd documentation
+<https://nsls-ii.github.io/ophyd/signals.html#kind/>`_. A quick inspection of
+the various attributes allows us to see how our signals are organized.
+
+.. ipython:: python
  
-   from qtpy.QtWidgets import QApplication
+    # Most important signal/s
```
Comment (klauer) [inline docs/source/basic_usage.rst]:
```suggestion
    # Most important signal(s)
```

File: docs/source/basic_usage.rst [hunk 3]
```diff
@@ -33,87 +33,153 @@ Interpreting a Device
...
-   app = QApplication()
+.. figure:: /_static/kind_panel.gif
+   :scale: 100%
+   :align: center
 
-   dg1_m1 = EpicsMotor('MFX:DG1:MMS:01', name="DG1 M1")
+Now, at first glance it may not be obvious, but there is a lot of information
+here! We know which of these signals an operator will want to control and which
+ones are purely meant to be read back. We also have these signals grouped by
+their importance to operation, each with a terse human legible description of
```
Comment (klauer) [inline docs/source/basic_usage.rst]:
```suggestion
their importance to operation, each with a terse human-readable description of
```
(legible feels off here)

File: docs/source/connections.rst [hunk 4]
```diff
@@ -47,3 +47,14 @@ Precision             `"precision"`
...
 Engineering Units     `"units"`
 ===================== ===============
+
+
+Happi Plugin
+============
+
+The PyDM Plugin interface makes no mandate about the type of signals
+that we connect to our widgets. The :class:`.HappiPlugin` and corresponding
+``HappiChannel`` reimagines theses signals and sends entire ``ophyd`` objects
```
Comment (klauer) [inline docs/source/connections.rst]: Not sure what `reimagines` is getting at?

File: README.md [hunk 5]
```diff
@@ -32,7 +32,7 @@ different categories based on their relevance to operators. Typhon uses this
 information to craft user interfaces.
 
 ## Installation
-Recommended installation:
+Recommended installation on Linux:
 ```
 conda install typhon -c pcds-tag -c conda-forge
```
Comment (ZLLentz) [inline README.md]: This wasn't changed in your PR, but this install line grabs other stuff from `pcds-tag` that the user will not want, such as our non-standard `pyqt` build. This is fine on-site, but annoying for people off-site. I recommend switching `conda-forge` to be first here.

I want to kill this build but I'm being dragged down by `pmgr`.

File: docs/source/basic_usage.rst (hunk 1 above)
Comment (teddyrendahl) [reply docs/source/basic_usage.rst]: It is a Jinja template and should be a string

File: README.md (hunk 5 above)
Comment (teddyrendahl) [reply README.md]: Astute point. Thanks

File: docs/source/connections.rst (hunk 4 above)
Comment (teddyrendahl) [reply docs/source/connections.rst]: Just that I'm not sending a value and instead an entire structure. Maybe tooting my own horn too much 😉

**Pull request merged**